lib/cli/cli.js

```diff
@@ -86,14 +86,21 @@ CLI.prototype.run = function(environment) {
       // Possibly this issue: https://github.com/joyent/node/issues/8329
       // Wait to resolve promise when running on windows.
       // This ensures that stdout is flushed so acceptance tests get full output
+      var result = exitCode;
+      if (this.testing) {
+        result = {
```

Comment:
@rwjblue @stefanpenner I just wanted to make sure this was okay before you merge, it doesn't seem to be breaking anything by returning this object instead of just the exitCode.